### ISSUE TITLE
Error handler broken again

### ISSUE THREAD

**Issue by nyuszika7h** [lykoss/lykos#163]
```
[2015-09-06 09:00:34+0000] <--- receive stark.coldfront.net notice (AUTH, *** Looking up your hostname...)
Traceback (most recent call last):
[2015-09-06 09:00:34+0000] closing socket
[2015-09-06 09:00:34+0000] Calling sys.exit()...
```

blame @Vgr255

**Comment by jacob1** [comment]
I tried fixing this but it wouldn't work. Then I removed the new error handler and it still wasn't working ...

So it isn't the new error handler's fault, probably we are catching oyoyo errors somewhere. I will look into it some other time.

**Comment by Vgr255** [comment]
Fixed in b28d4bf
